### PR TITLE
Adding more comments

### PR DESCRIPTION
**Pull request by Fourchaux**
La Saga Commentaires continue ...

### CODE REVIEW

File: src/core/CCArrayLabels.mli [hunk 1]
```diff
@@ -129,23 +132,23 @@ val lookup : cmp:'a ord -> key:'a -> 'a t -> int option
...
-(** Same as {!lookup}, but
+(** Like {!lookup}, but
     @raise Not_found if the key is not present. *)
 
 val bsearch : cmp:('a -> 'a -> int) -> key:'a -> 'a t ->
   [ `All_lower | `All_bigger | `Just_after of int | `Empty | `At of int ]
 (** [bsearch ?cmp key arr] finds the index of the object [key] in the array [arr],
     provided [arr] is {b sorted} using [cmp]. If the array is not sorted,
-    the result is not specified (may raise Invalid_argument).
+    the result is not specified (may @raise Invalid_argument).
```
Comment (c-cube) [inline src/core/CCArrayLabels.mli]: this is probably invalid ocamldoc. Have you tried compiling doc?

File: src/core/CCArray.mli [hunk 2]
```diff
@@ -70,7 +74,7 @@ val fold_map : ('acc -> 'a -> 'acc * 'b) -> 'acc -> 'a t -> 'acc * 'b t
 
 val scan_left : ('acc -> 'a -> 'acc) -> 'acc -> 'a t -> 'acc t
 (** [scan_left f acc a] returns the array
-    [ [|acc; f acc x0; f (f acc a.(0)) a.(1); …|] ].
+    [ [|acc; f acc x0; f (f acc a.(0)) a.(1); ...|] ].
```
Comment (c-cube) [inline src/core/CCArray.mli]: come on, is that really necessary? 😆

File: src/core/CCFloat.mli [hunk 3]
```diff
@@ -13,25 +13,39 @@ type fpclass = Pervasives.fpclass =
...
+(** Positive infinity. Equal to {!Pervasives.infinity}. *)
+
 val min_value : t
+(** Negative infinity. Equal to {!Pervasives.neg_infinity}. *)
 
 val max_finite_value : t
+(** Equal to {!Pervasives.max_float}. *)
 
 val epsilon : t
+(** The smallest positive float x such that [1.0 +. x <> 1.0].
```
Comment (c-cube) [inline src/core/CCFloat.mli]: 👏

File: src/core/CCFormat.mli [hunk 4]
```diff
@@ -84,48 +84,48 @@ val within : string -> string -> 'a printer -> 'a printer
...
     @since 0.16 *)
 
 val return : ('a, _, _, 'a) format4 -> unit printer
 (** [return "some_format_string"] takes a argument-less format string
     and returns a printer actionable by [()].
     Examples:
-    - [return ",@ "]
-    - [return "@{<Red>and then@}@,"]
-    - [return "@[<v>a@ b@]"]
+    - [return ",@ "].
```
Comment (c-cube) [inline src/core/CCFormat.mli]: I'd rather not put dots here, it'll look like it's part of the code snippet

File: src/core/CCOrd.mli [hunk 5]
```diff
@@ -60,9 +60,15 @@ val map : ('a -> 'b) -> 'b t -> 'a t
       first component. *)
 
 val (>|=) : 'b t -> ('a -> 'b) -> 'a t
-(** Infix equivalent of {!map} *)
+(** Infix equivalent of {!map}. *)
 
 module Infix : sig
   val (<?>) : int -> ('a t * 'a * 'a) -> int
+  (** [c1 <?> (ord, x, y)] returns the same as [c1] if [c1] is not [0];
```
Comment (c-cube) [inline src/core/CCOrd.mli]: nice.

File: src/core/CCArray.mli (hunk 2 above)
Comment (Fourchaux) [reply src/core/CCArray.mli]: 55+ dotdotdot are already present in the code.
They must be modified?

```
Also, by trying 

$ ocamldoc -html CCArray.mli

I've got (with  CCArray.html):

 ... [|acc; f acc x0; f (f acc a.(0)) a.(1); â€¦|] 
```

```

File: src/core/CCFormat.mli (hunk 4 above)
Comment (Fourchaux) [reply src/core/CCFormat.mli]: OK. No dots.

File: src/core/CCArrayLabels.mli (hunk 1 above)
Comment (Fourchaux) [reply src/core/CCArrayLabels.mli]: Effectively, this is incorrect with ocamldoc. (no more printing after may).

@ deleted.

File: src/core/CCArray.mli (hunk 2 above)
Comment (Fourchaux) [reply src/core/CCArray.mli]: 55+ dotdotdot are already used in the code.

**Pull request merged**